test/uk/gov/hmrc/gform/core/parsers/FormatParserSpec.scala

```diff
@@ -80,6 +83,17 @@ class FormatParserSpec extends Spec {
                          |after 2016-6-9 9           ^""".stripMargin))
   }
 
+  "after YYYY-04-DD" should "throw exception" in {
```

Comment:
I don't understand the intention of this test.

Reply:
With abstract dates we decided that it only made sense for it to work with precisely, rather than before and after. This was to test that an abstract date did not get parsed with "after" before it. Thanks for your changes lance I'll get the done either tonight or tomorrow